includes/admin/class-wc-admin-addons.php

```diff
@@ -287,9 +287,11 @@ public static function output_small_dark_block( $block ) {
 			<div class="addons-small-dark-items">
 				<?php foreach ( $block->items as $item ) : ?>
 					<div class="addons-small-dark-item">
-						<div class="addons-small-dark-item-icon">
-							<img class="addons-img" src="<?php echo esc_url( $item->image ); ?>" />
-						</div>
+						<?php if ( $item->image ) : ?>
+							<div class="addons-small-dark-item-icon">
+								<img class="addons-img" src="<?php echo esc_attr( $item->image ); ?>" />
```

Comment:
Fixed in: https://github.com/woocommerce/woocommerce/pull/12012/commits/f740497883eee6e2ff4ecc0dcc505f364e7e0770